OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -123,6 +124,11 @@ public void InvokeCommand(string name, string arg)
 					}
 
 					break;
+
+				case "poweroutage":
+					foreach (var player in world.Selection.Actors.Select(a => a.Owner.PlayerActor).Distinct())
+						player.Trait<PowerManager>().TriggerPowerOutage(125);
```

Comment:
```
Exception of type `System.InvalidOperationException`: CheckSyncUnchanged: sync-changing code may not run here
```

This needs to be routed through an order instead.  You should also turn on "Check sync around unsynced code" in the advanced options.